src/main/java/io/opentelemetry/javaagent/BytesAndName.java

```diff
@@ -1,12 +0,0 @@
-/* (C)2021 */
```

Comment:
Czemu się tych plików pozbywamy?

Reply:
Nie wiem jak to się stało, przez przypadek chyba, zaraz je dodam z powrotem